Alerting docs: Update Intro - alert rule labels and annotations (#88230)
grafana/grafana: docs/sources/alerting/fundamentals/alert-rules/annotation-label.md

      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/notifications/templates/
  alert-rule-evaluation:
    - pattern: /docs/grafana/
      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/alert-rule-evaluation/
    - pattern: /docs/grafana-cloud/
      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/alert-rule-evaluation/
  silences:
    - pattern: /docs/grafana/
      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/create-silence/
    - pattern: /docs/grafana-cloud/
      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/create-silence/
  notification-policies:
    - pattern: /docs/grafana/
      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/notifications/notification-policies/
    - pattern: /docs/grafana-cloud/
      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/notifications/notification-policies/
---

# Labels and annotations

Labels and annotations add additional information about an alert using key/value pairs:

- [Labels](#labels) are used to differentiate an alert from all other alerts.

> - [Annotations](#annotations) are used to provide extra details to an existing alert.

Reviewer: ```suggestion
- [Annotations](#annotations) are used to provide extra detail on an existing alert.
```

Data source query labels labels are also used to generate multiple alert instances from the same alert rule, helping to distinguish alerts from different data.

**Reserved labels**

Reserved labels are automatically added by Grafana:

- `alert_name`: the name of the alert rule.
- `grafana_folder`: the title of the folder containing the alert.

Labels prefixed with `grafana_` are reserved by Grafana for special use. To stop Grafana Alerting from adding a reserved label, you can disable it via the `disabled_labels` option in [unified_alerting.reserved_labels](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#unified_alertingreserved_labels) configuration.

{{<admonition type="note">}}

Two alert rules cannot produce alert instances with the same labels. If two alert rules have the same labels such as `foo=bar,bar=baz` and `foo=bar,bar=baz` then one of the generated alert instances will be discarded.

Ensure the label set for an alert does not have two or more labels with the same name.

> - If a configured label has the same name as a data source query label, it will replace the data source label.

Reviewer: ```suggestion
- If a configured label has the same name as a data source query label, it replaces the data source label.
```

> - If a configured label has the same name as a reserved label, it will be omitted.

Reviewer: ```suggestion
- If a configured label has the same name as a reserved label, it is omitted.
```
